tracer.go

```diff
@@ -327,8 +327,7 @@ type Tracer struct {
 	breakdownMetrics  *breakdownMetrics
 	profileSender     profileSender
 
-	statsMu sync.Mutex
-	stats   TracerStats
+	stats TracerStats
```

Comment:
We should probably heap allocate this, so ensure its fields are properly aligned for atomic access. Otherwise changes in fields above may cause it to misalign on 32-bit architectures (if it isn't already).

Reply:
Wasn't aware of the atomic bug, I've moved it to a pointer type in the Tracer.